yb-voyager/src/anon/sql_anonymizer.go

```diff
@@ -2233,65 +2151,119 @@ func (a *SqlAnonymizer) anonymizeColumnDefNode(cd *pg_query.ColumnDef) (err erro
 			continue
 		}
 
-		// Handle A_Const nodes (literal values)
-		aConst := constraint.RawExpr.GetAConst()
-		if aConst == nil || aConst.Val == nil {
-			continue
+		// Handle constant values in default values using the consolidated function
+		if err := a.anonymizeConstantValue(constraint.RawExpr, CONST_KIND_PREFIX); err != nil {
+			return fmt.Errorf("anon coldef default value: %w", err)
 		}
+	}
 
-		// Handle string literals in default values
-		if sval := aConst.GetSval(); sval != nil {
-			sval.Sval, err = a.registry.GetHash(CONST_KIND_PREFIX, sval.Sval)
-			if err != nil {
-				return fmt.Errorf("anon coldef default string value: %w", err)
-			}
-			continue
+	return nil
+}
+
+// anonymizeConstantValue handles anonymization of constant values in SQL statements
+// This function consolidates the logic for handling different types of constant values
+// including TypeName, A_Const (string, int, float, bool), String_, and raw values
+func (a *SqlAnonymizer) anonymizeConstantValue(arg *pg_query.Node, prefix string) error {
```

Comment:
nit: `anonymizeArgConstantValue`

Reply:
didn't get this. I thought of having this function as a generic for any constant value anonymization

Reply:
okay, then change the first parameter name to `node`